extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -153,7 +162,7 @@ describes.repeated('', {
       });
 
       it('should server side render templates if enabled', () => {
-        return getAmpForm(getForm()).then(ampForm => {
+        ampForm.then(ampForm => {
```

Comment:
You still want to return the Promise here and above so the test knows when it's done

Reply:
done. thanks for the prompt review.